File: server-e/app/Console/Commands/HealthHeartbeat.php
<?php

namespace App\Console\Commands;

use App\Services\HealthReporter;
use Illuminate\Console\Command;

class HealthHeartbeat extends Command
{
    protected $signature = 'health:heartbeat';

    protected $description = 'Send Server E heartbeat to the Alert Server';

    public function handle(
        HealthReporter $healthReporter
    ): int {
        $success = $healthReporter->heartbeat();

        if ($success) {
            $this->info('Heartbeat sent successfully.');

            return self::SUCCESS;
        }

        $this->error('Failed to send heartbeat.');

        return self::FAILURE;
    }
}

File: server-d/api/management/commands/health_heartbeat.py
from django.core.management.base import BaseCommand

from api.services.health_reporter import HealthReporter


class Command(BaseCommand):

    help = "Send Server D heartbeat to the Alert Server"

    def handle(self, *args, **options):

        reporter = HealthReporter()

        success = reporter.heartbeat()

        if success:
            self.stdout.write(
                self.style.SUCCESS(
                    "Heartbeat sent successfully."
                )
            )

            return

        self.stdout.write(
            self.style.ERROR(
                "Failed to send heartbeat."
            )
        )